static/styles/input_pill.css

```diff
@@ -32,6 +32,20 @@
     outline: none;
 }
 
+.notmem {
+    cursor: not-allowed;
+    border: none;
+}
+
+.notmem .pill {
+    cursor: not-allowed;
+}
+
+.notmem .pill:focus {
+    color: hsl(177, 38%, 54%);
+    border: 1px solid #b2e4e2;
+}
```

Comment:
These should be scoped inside `.pill-container` -- we don't like having CSS that by appearance could affect other parts of the site.